runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -234,9 +220,14 @@ public void onGameTick(GameTick event)
 			notifier.notify("[" + local.getName() + "] is about to log out from being online for 6 hours!");
 		}
 
-		if (config.animationIdle() && checkAnimationIdle(waitDuration, local))
+		if (checkAnimationIdle(waitDuration, local))
 		{
-			notifier.notify("[" + local.getName() + "] is now idle!");
+			if (currentSkill.equals("Herblore") || currentSkill.equals("Other") || currentSkill.equals("Prayer"))
+			{
+				notifier.notify("[" + local.getName() + "] is now idle");
+				return;
+			}
+			notifier.notify("[" + local.getName() + "] is no longer " + currentSkill);
```

Comment:
Just add "traning x" and no need for special casing

Reply:
This has been generalized without adding training to everything because that looks ugly